File: TravelBois2/WebApp/ClientApp/src/app/entities/objects/brza-rezervacija.ts
export class BrzaRezervacija {
  idLeta: number;
  cenaSedista: number;
  idSedista: string;

  constructor(idLeta: number, idSedista: string, cenaSedista: number) {
    this.idLeta = idLeta;
    this.idSedista = idSedista;
    this.cenaSedista = cenaSedista;
  }
}

File: TravelBois2/WebApp/Models/BrzaRezervacija.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace WebApp.Models
{
    public class BrzaRezervacija
    {
        [Key]
        [Required]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        [Required]
        public int IdLeta { get; set; }
        [Required]
        public string IdSedista { get; set; }
        [Required]
        public int CenaSedista { get; set; }
    }
}
